libmcount/mcount.c

```diff
@@ -449,6 +449,85 @@ enum filter_result mcount_entry_filter_check(struct mcount_thread_data *mtdp,
 	return FILTER_IN;
 }
 
+static int script_save_context(struct script_context *sc_ctx,
+			       struct mcount_thread_data *mtdp,
+			       struct mcount_ret_stack *rstack,
+			       struct sym *sym, char *symname,
```

Comment:
I think the `sym` parameter can be omitted.

Reply:
Oops.. will fix it